src/workers/commentMentionEmail.ts

```diff
@@ -0,0 +1,53 @@
+import { FastifyLoggerInstance } from 'fastify';
+import { CommentMention } from './../entity/CommentMention';
+import { Comment, User } from '../entity';
+import { pickImageUrl } from '../common';
+import { baseNotificationEmailData, sendEmail, truncatePost } from '../common';
+import { Connection } from 'typeorm';
+import { getPostPermalink } from '../schema/posts';
+
+const getContent = (comment: string, username: string) => {
+  const mention = `@${username}`;
+
+  if (comment === mention) {
+    return '';
+  }
+
+  const [firstWord, ...content] = comment.split(' ');
+
+  return firstWord === mention ? content.join(' ') : comment;
+};
+
+export const sendEmailToMentionedUser = async (
+  con: Connection,
+  commentMention: CommentMention,
+  logger: FastifyLoggerInstance,
+): Promise<void> => {
+  const comment = await con
+    .getRepository(Comment)
+    .findOne(commentMention.commentId);
+  const post = await comment.post;
+  const commenter = await comment.user;
+  const mentioned = await con
+    .getRepository(User)
+    .findOne(commentMention.mentionedUserId);
+  const [first_name] = mentioned.name.split(' ');
+  const content = getContent(comment.content, mentioned.username);
+
+  await sendEmail({
+    ...baseNotificationEmailData,
+    to: mentioned.email,
+    templateId: 'd-6949e2e50def4c6698900032973d469b',
```

Comment:
We might want to find a way to make this fetch from the config or something?
I'm thinking what happens if we want to change template, but maybe it's overkill.
Let me know what you guys think.